app/Helpers/DatabaseHelper.php

```diff
@@ -7,41 +7,41 @@
 use App\Models\GamesFile;
 use Carbon\Carbon;
 
-class DatabaseHelper{
+class DatabaseHelper {
 
-    public static function getReleaseDateFromGameId($gameid){
+    public static function getReleaseDateFromGameId($gameid) {
         $game = Game::whereId($gameid)->first();
 
-        if(is_null($game)){
+        if (is_null($game)) {
             return '';
         }
 
-        if(Carbon::parse($game->release_date)->year == -1 || is_null($game->release_date)){
+        if (Carbon::parse($game->release_date)->year == -1 || is_null($game->release_date)) {
             $releasedate = GamesFile::whereGameId($gameid)
                 ->selectRaw('CONCAT(release_year, "-", LPAD(release_month, 2, "0"), "-", release_day) as reldate')
                 ->orderBy('release_type', 'desc')
                 ->orderBy('reldate', 'asc')
                 ->first();
 
-            if($releasedate){
+            if ($releasedate) {
                 return Carbon::parse($releasedate->reldate)->toDateString();
-            }else{
+            }else {
                 return '';
             }
 
-        }else{
+        }else {
             return Carbon::parse($game->release_date)->toDateString();
         }
     }
 
-    public static function getDevelopersUrlList($gameid){
+    public static function getDevelopersUrlList($gameid) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->